src/allencell_ml_segmenter/main/main_widget.py

```diff
@@ -169,6 +174,16 @@ def _handle_new_model(self, _: Event) -> None:
             else self._prediction_view
         )
 
+    def _handle_existing_model(self, _: Event) -> None:
+        """
+        Handle existing model selection (disable tabs).
+
+        inputs:
+            is_new_model - bool
+        """
+        self._window_container.setTabEnabled(0, False)
```

Comment:
Dont we have a mechanism already switching between tabs?

Reply:
I am thinking of this https://github.com/AllenCell/allencell-segmenter-ml/blob/main/src/allencell_ml_segmenter/main/main_model.py#L48

Reply:
I use this mechanism to switch tabs already. this is just to disable the tabs after training completes. I'll rename for clarity.

Reply:
https://github.com/AllenCell/allencell-segmenter-ml/blob/852b8c8e6894bc71f9e85c53e0cce7f88eef6998/src/allencell_ml_segmenter/prediction/view.py#L109-L113

Reply:
Shouldnt disabling / enabling be part of the same mechanism as selecting a tab?  (is the behaviors of selecting bound with disabling the other tabs?)

Reply:
https://github.com/AllenCell/allencell-segmenter-ml/issues/523